### PR TITLE
My first change in Daniels repo

### PR DESCRIPTION
**Pull request by hollatz**
I added a python file that represents my coding skills.

### CODE REVIEW

File: src/Dominiks_modification.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print('Dominik is great!')
```
Comment (danielseipt) [inline src/Dominiks_modification.py:1]: Is he?

Comment (danielseipt) [reply src/Dominiks_modification.py:1]: Where's the new features of the the code?

**Comment by danielseipt** [comment]
Why you don't see my comments?

File: src/Dominiks_modification.py (hunk 1 above)
Comment (maxbalrog) [inline src/Dominiks_modification.py:1]: Warum ist es so einfach????

Comment (danielseipt) [inline src/Dominiks_modification.py:1]: What does this do?

**Comment by hollatz** [comment]
Here is my comment.

**Review by danielseipt — CHANGES_REQUESTED** [review]

File: src/Dominiks_modification.py (hunk 1 above)
Comment (hollatz) [reply src/Dominiks_modification.py:1]: How do I change this?

**Review by hollatz — COMMENTED** [review]
OK never mind.

**Comment by hollatz** [comment]
This closed issue #4

**Pull request merged**